fix(learn/css): update replaced elements in grid layout description (#34317)
mdn/content: files/en-us/learn/css/building_blocks/images_media_form_elements/index.md

## Replaced elements in layout

> When using various CSS layout techniques on replaced elements, you may well find that they behave slightly differently from other elements. For example, in a grid layout, elements are stretched by default to fill the entire [grid areas](/en-US/docs/Glossary/Grid_Areas) in which they lie in. Images do not stretch, and instead they are aligned to the start of the corresponding grid areas.

Reviewer: ```suggestion
When using various CSS layout techniques on replaced elements, you may well find that they behave slightly differently from other elements. For example, in a grid layout, elements are stretched by default to fill their entire [grid areas](/en-US/docs/Glossary/Grid_Areas). Images do not stretch; instead, they are aligned to the start of their grid areas.
```
